src/plugins/tagfilter/tagfilter-en.hbs

```diff
@@ -981,6 +982,246 @@
 &lt;/section></code></pre>
 </details>
 
+
+<section id="filteredFinder" class="wb-tagfilter provisional wb-filter" data-wb-tagfilter='{ "live": false }' data-wb-filter='{
+		"selector": "section",
+		"section": ".wb-tagfilter-items",
+		"uiTemplate": "#searchFilter2",
+		"live": false
+	}'>
+	<h2>Example 4 - Finder with "or" behavior updating with a button</h2>
+
+	<div class="row">
+	<search id="searchFilter2" class="col-xs-12 col-md-3 pull-right" aria-label="On this page">
```

Comment:
Why there is an `aria-label` here? There is already a few heading preceding and inside the search section.

Reply:
Very fair point, there is likely already enough context here without the aria-label. Change was made with new commit.

Was just making sure it wasn't conflicting with the site-wide search element which also has a aria-role="search".